tests/UI/utils/date.ts

```diff
@@ -53,4 +53,32 @@ export default {
         throw new Error(`The format ${format} is not handled by this helper yet`);
     }
   },
+
+  /**
+   * Set dateTime format
+   * @param format {string} Format of the date
+   * @param dateTime {string} Date time to change format
+   * @returns {string}
+   */
+  setDateFormat(format: string, dateTime: string): string {
```

Comment:
There is not much difference between this and `getDateFormat`. what's the difference ? can we make it one function ?

Reply:
The difference is that `getDateFormat` Creates a date format from the string 'past, feature and present' but this function creates a date format from a dateTime and transforms it into the desired format.